docs/subinterpreters.md

```diff
@@ -1,7 +1,7 @@
 # RedisGears Design: Python Sub-Interpreters
 RedisGears ships with an embedded [Python Interpreter](runtime.md#python-interpreter). This makes it possible to run Python code using the [`RG.PYEXECUTE` command](commands.md#rgpyexecute). Because the interpreter is a singleton that's shared among all calls to `RG.PYEXECUTE`, there's the risk of different executions using the same identifiers (e.g. global variables, functions or class names).
 
-One way to address this problem is by restarting the interpreter before each execution. However, an interpreter restart is a time-costly operation that renders this approach less-than-desirable. Also, two executions may be parallel, either in two different threads or even in the same thread (while the execution of the Python code itself is always non-parallelized, the rest of the execution's lifecycle may be). When more than one execution is running, restarting the interpreter is no longer an option. To address the issue correctly RedisGears make use of [Python's Sub-Interpreters](https://docs.python.org/3/c-api/init.html#sub-interpreter-support).
+One way to address this problem is by restarting the interpreter before each execution. However, an interpreter restart is a time-costly operation that renders this approach less-than-desirable. Also, two executions may be parallel, either in two different threads or even in the same thread (while the execution of the Python code itself is always non-parallelized, the rest of the execution's lifecycle may be). When more than one execution is running, restarting the interpreter is no longer an option. To address the issue correctly RedisGears plans to make use of [Python's Sub-Interpreters](https://docs.python.org/3/c-api/init.html#sub-interpreter-support).
```

Comment:
Maybe we should mentioned that currently we solve it with new global dictionary for each pyexecute (because we found out that there are libraries which are not compatible with subinyerpreters). Its not perfect but gives a good isolation and we are planning to return subinterpreters in the future?

Reply:
So repeat some of the stuff from https://github.com/RedisGears/RedisGears/blob/master/docs/runtime.md#python-interpreter - makes sense. The current is simply not true so we need to resolve it :)

Reply:
Yes I agree, we should call this page "runs isolation" or something and explain how its done today and what are the future plans. If you want I can take it.